packages/turf-transform-scale/index.js

```diff
@@ -59,18 +61,24 @@ module.exports = function (geojson, factor, origin, mutate) {
  * @param {Feature|Geometry} geojson GeoJSON Feature/Geometry
  * @param {number} factor of scaling, positive or negative values greater than 0
  * @param {string|Geometry|Feature<Point>|Array<number>} [origin="centroid"] Point from which the scaling will occur (string options: sw/se/nw/ne/center/centroid)
+ * @param {boolean} [mutate=false] allows GeoJSON input to be mutated
  * @returns {Feature|Geometry} scaled GeoJSON Feature/Geometry
  */
-function scale(geojson, factor, origin) {
+function scale(geojson, factor, origin, mutate) {
     // Default params
-    var isPoint = (geojson.type === 'Point' || geojson.geometry && geojson.geometry.type === 'Point');
+    var geomType = getGeomType(geojson);
+    var isPoint = geomType === 'Point';
+    var isPolygon = geomType === 'Polygon' || geomType === 'MultiPolygon';
     origin = defineOrigin(geojson, origin);
 
     // Shortcut no-scaling
     if (factor === 1 || isPoint) return geojson;
 
     // Scale each coordinate
-    coordEach(geojson, function (coord) {
+    coordEach(geojson, function (coord, coordIndex, featureIndex, featureSubIndex) {
+        // Ignore scaling on first coordinate of Polygons (Issue #895)
+        if (isPolygon && mutate === true && featureSubIndex === 0) return;
```

Comment:
@stebogit Might want to add this type of behavior in a few of the `transform` modules.